wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/Workspace.java

```diff
@@ -38,16 +39,20 @@ public class Workspace {
   // this is for insights page in UI
   private final JsonObject insights;
 
+  private HashMap<String, UserDefinedAction> nullabilityMap;
```

Comment:
HashMap -> Map (same for the rest of the class)

Reply:
this should also be final

Reply:
Done